Scenarioo/Scenarioo/Model/Docu/Entities/ScreenAnnotation.cs

```diff
@@ -0,0 +1,95 @@
+﻿/* scenarioo-api
+ * Copyright (C) 2014, scenarioo.org Development Team
+ * 
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * As a special exception, the copyright holders of this library give you 
+ * permission to link this library with independent modules, according 
+ * to the GNU General Public License with "Classpath" exception as provided
+ * in the LICENSE file that accompanied this code.
+ * 
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * 
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+using System;
+using System.Xml.Serialization;
+
+using Scenarioo.Model.Docu.Entities.Generic;
+
+namespace Scenarioo.Model.Docu.Entities
+{
+    [Serializable]
+    [XmlRoot("screenAnnotation")]
+    public class ScreenAnnotation
+    {
+        [XmlElement("region")]
+        public ScreenRegion Region { get; set; }
+
+        /// <summary>
+        /// (optional) Set the visual style of the annotation (if not set, the same style as 'neutral' will be used).
```

Comment:
@tobiaszuercher the style 'neutral' that is mentioned in this comment, does not exist anymore. This is an outdated comment (from me), that you probably copied from the Java library. Instead 'default' is the default. I will fix that in the Java library. Can you please fix it here?